includes/wp-cli.php

```diff
@@ -8,96 +8,152 @@
 class WP_Stream_WP_CLI_Command extends WP_CLI_Command {
 
 	/**
-	 * Subcommand to query a set of Stream records
+	 * Query a set of Stream records.
 	 *
-	 * You may use any associative argument(s) available in the
-	 * WP_Stream_Query::instance()->query() method.
+	 * ## OPTIONS
 	 *
-	 * Examples:
-	 * wp stream query --author_role__not_in=administrator --date_after=2015-01-01T12:00:00
-	 * wp stream query --author=1 --action=login --records_per_page=50 --fields=created
+	 * [--fields=<fields>]
+	 * : Limit the output to specific object fields.
+	 *
+	 * [--<field>=<value>]
+	 * : One or more args to pass to wp_stream_query.
+	 *
+	 * ## AVAILABLE FIELDS TO QUERY
+	 *
+	 * You can build a query from these fields:
+	 *
+	 * * author
+	 * * author__in
+	 * * author__not_in
+	 * * author_role
+	 * * author_role__in
+	 * * author_role__not_in
+	 * * date
+	 * * date_from
+	 * * date_to
+	 * * date_after
+	 * * date_before
+	 * * ip
+	 * * ip__in
+	 * * ip__not_in
+	 * * connector
+	 * * connector__in
+	 * * connector__not_in
+	 * * context
+	 * * context__in
+	 * * context__not_in
+	 * * action
+	 * * action__in
+	 * * action__not_in
+	 * * search
+	 * * search_field
+	 * * record
+	 * * record__in
+	 * * record__not_in
+	 * * records_per_page
+	 * * paged
+	 * * order
+	 * * orderby
+	 *
+	 * ## AVAILABLE FIELDS
+	 *
+	 * These fields will be displayed by default for each post:
+	 *
+	 * * created
+	 * * ip
+	 * * author
+	 * * author_meta.user_login
+	 * * author_role
+	 * * summary
+	 *
+	 * These fields are optionally available:
+	 *
+	 * * ID
+	 * * site_id
+	 * * blog_id
+	 * * object_id
+	 * * connector
+	 * * context
+	 * * action
+	 * * author_meta
+	 * * stream_meta
+	 * * meta.links.self
+	 * * meta.links.collection
+	 * * meta.score
+	 * * meta.sort
+	 *
+	 * ## EXAMPLES
+	 *
+	 *     wp stream query --author_role__not_in=administrator --date_after=2015-01-01T12:00:00
+	 *     wp stream query --author=1 --action=login --records_per_page=50 --fields=created
 	 *
 	 * @see WP_Stream_Query
-	 * @subcommand query
+	 * @see https://github.com/wp-stream/stream/wiki/Query-Reference
 	 */
 	public function query( $args, $assoc_args ) {
-		self::connection();
+		$start             = microtime( true );
+		$query_args        = array();
+		$formatted_records = array();
 
-		$start      = microtime( true );
-		$query_args = array();
+		$this->connection();
 
-		foreach ( $assoc_args as $key => $value ) {
-			$query_args[ $key ] = $value;
+		if ( empty( $assoc_args['fields'] ) ) {
+			$fields = array( 'created', 'ip', 'author', 'author_meta.user_login', 'author_role', 'summary' );
```

Comment:
Hey @szepeviktor, sorry for my absence - I just returned from paternity leave. Yeah, I think a separate wiki page for WP-CLI makes sense for that. Just created this placeholder: https://github.com/wp-stream/stream/wiki/WP-CLI-Command